.gitignore

```diff
@@ -1,3 +1,9 @@
+# Ignore Gradle project-specific cache directory
+.gradle
+
+# Ignore Gradle build output directory
+build
```

Comment:
`build` appears twice as well as 'build/' do we need them all?

Reply:
Thanks, it looks like a single `build/` is just enough.